.gitignore

```diff
@@ -0,0 +1,4 @@
+**pycache**
+db.json
+.wayscript
+*.http
```

Comment:
What's a .http file?

Reply:
I was using it to test the APIs using a vs code extension. It required a `.http` file to define the request tests.